# calib information
float64 fx
float64 fy
float64 cx
float64 cy
std_msgs/Float32MultiArray value_scaledf   # [fx, fy, cx, cy] in float
std_msgs/Float32MultiArray value_scaledi   # inverse of value_scaledf